frontend/src/load-bar.js

```diff
@@ -40,9 +40,6 @@ const init = () => {
 };
 
 const loadBarFactory = () => {
-  const _loader = null;
-  let _currentJobId = null;
```

Comment:
Why delete these two var ?

Reply:
They were listed as not used by typescript, so I removed them without thinking. And indeed, const & let are not assigned to `this`.